apps/backoffice-v2/src/lib/blocks/hooks/useKybRegistryInfoBlock/useKybRegistryInfoBlock.tsx

```diff
@@ -1,11 +1,38 @@
 import { useMemo } from 'react';
 import { createBlocksTyped } from '@/lib/blocks/create-blocks-typed/create-blocks-typed';
+import { WarningFilledSvg } from '@/common/components/atoms/icons';
+import * as React from 'react';
 
-export const useKybRegistryInfoBlock = ({ pluginsOutput, workflow }) => {
-  return useMemo(() => {
-    if (Object.keys(pluginsOutput?.businessInformation?.data?.[0] ?? {}).length === 0) {
-      return [];
-    }
+export const useKybRegistryInfoBlock = ({ pluginsOutput, workflow }) =>
+  useMemo(() => {
+    const cell = Object.keys(pluginsOutput?.businessInformation?.data?.[0] ?? {}).length
+      ? {
+          type: 'details',
+          hideSeparator: true,
+          value: {
+            data: Object.entries(pluginsOutput?.businessInformation?.data?.[0])?.map(
+              ([title, value]) => ({
+                title,
+                value,
+              }),
+            ),
+          },
+          workflowId: workflow?.id,
+          documents: workflow?.context?.documents,
+        }
+      : {
+          type: 'paragraph',
+          value: (
+            <span className="mt-[20px] flex text-sm text-black/60">
+              <WarningFilledSvg
+                className={'mr-[8px] mt-px text-black/20'}
+                width={'20'}
+                height={'20'}
+              />
+              <span>{pluginsOutput?.businessInformation?.message}</span>
+            </span>
+          ),
+        };
```

Comment:
Feel free to create a type helper or alias. This is only needed until the blocks package is fixed.
```suggestion
    const cell = Object.keys(pluginsOutput?.businessInformation?.data?.[0] ?? {}).length
      ? {
          type: 'details',
          hideSeparator: true,
          value: {
            data: Object.entries(pluginsOutput?.businessInformation?.data?.[0])?.map(
              ([title, value]) => ({
                title,
                value,
              }),
            ),
          },
          workflowId: workflow?.id,
          documents: workflow?.context?.documents,
        } satisfies Extract<Parameters<ReturnType<typeof createBlocksTyped>['addCell']>[0], {
            type: 'details';
          }>
      : {
          type: 'paragraph',
          value: (
            <span className="mt-[20px] flex text-sm text-black/60">
              <WarningFilledSvg
                className={'mr-[8px] mt-px text-black/20'}
                width={'20'}
                height={'20'}
              />
              <span>{pluginsOutput?.businessInformation?.message}</span>
            </span>
          ),
        } satisfies Extract<Parameters<ReturnType<typeof createBlocksTyped>['addCell']>[0], {
            type: 'paragraph';
          }>;
```